### PR TITLE
Beautify and clean up CSS

### PR DESCRIPTION
**Pull request by anewb87**
#### What's this PR do?
Beautify and clean up CSS
- made sizes and margins more uniform
- lowercased all words
- general beautification

#### Where should the reviewer start?
`npm start` simply view the page, hover over the buttons

#### How should this be manually tested?
See above

#### Any background context you want to provide?
No

#### What are the relevant tickets?
NA

#### Screenshots (if appropriate)
NA

#### Questions:
Nope

### CODE REVIEW

**Review by kammon10 — APPROVED** [review]
its so pretty!!!!!

**Review by cagallo — APPROVED** [review]
Omg this is beautiful, love the change to lower case!

File: src/index.html [hunk 1]
```diff
@@ -6,123 +6,123 @@
...
+      <button class="hidden">home</button>
+      <button>cookbook</button>
+      <button>favorite recipes</button>
     </div>
   </nav>
   <main>
     <div class="flex-row main-view-container main-view-container-js">
       <section class="filter-section filter-section-js">
-        <h2>Filter</h2>
+        <h3>imma lookin'</h3>
```
Comment (aliroemhildt) [inline src/index.html:27]: LOVE THIS

File: src/index.html [hunk 2]
```diff
@@ -6,123 +6,123 @@
...
         <div class="filters">
           <div class="single-filter">
             <label for="antipasti">
-              <input class="filter-checkbox tag" type="checkbox" id="antipasti" value="yes">Antipasti
+              <input class="filter-checkbox tag" type="checkbox" id="antipasti" value="yes">antipasti
```
Comment (aliroemhildt) [inline src/index.html:31]: curious to see what the "yes" value does! so much info in one small input element

File: src/styles.css [hunk 3]
```diff
@@ -132,7 +135,8 @@ h2 {
 
 /* ~~~~~~~~~~ RECIPE CARD SECTION ~~~~~~~~~~~~ */
 .recipe-card {
-  border: 2px solid #1B3231;
+  /* border: 2px solid #1B3231; */
```
Comment (aliroemhildt) [inline src/styles.css:138]: we can probable delete this commented out line! unless you are saving it for something/still deciding, in that case never mind

**Pull request merged**